### ISSUE TITLE
Notification alert on whatsapp for anyother app

### ISSUE THREAD

**Issue by amarnath-jena** [Cacti/cacti#3204]
## Problem

Our team receives too many email notifications from Cacti.

## Requested solution

Support notifications through an application such as WhatsApp, or document an appropriate alternative notification integration.

**Comment by bmfmancini** [comment]
On thold enable command execution then do a curl to a twilo type service
for sms
Simple script to do it

Unfortunately for up and down your still stick to email until the command
execution feature shows on in the monitor plugin and cacto core
I have a feature request in for that

**Comment by netniV** [comment]
So many requests... if only we had a penny for each one, we'd be... well, pennies richer. ;-)

But yes, as Sean has stated, the feature doesn't exist in core yet.  There is talk about bringing it to core with 1.3, though with recent CVE's over running unexpected programs, we need to be very cautious and specific about how we handle things.

**Comment by bmfmancini** [comment]
Yes agree especially with command execution in APP could be opening some doors 
@netniV  could we move the routine that THOLD uses for command execution over to the Monitor plugin? I mean the THOLD command execution seems to work great
